sale_partner_selectable_option/models/sale_order.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.model
+    def fields_view_get(
```

Comment:
This is deprecated:

https://github.com/OCA/sale-workflow/actions/runs/4822025773/jobs/8588705236?pr=2495#step:8:117

Reply:
Changed using as reference:  https://github.com/odoo/odoo/commit/b03c227e885efa4ccdcad43ccb56ee10371b9284#